src/Business/Detection/Common/Models/PDetection.cs

```diff
@@ -0,0 +1,18 @@
+namespace Business.Detection.Common.Models
+{
+    using System;
+    using Business.Pagination;
+
+    public class PDetection
+    {
+        public int Id { get; set; }
+
+        public int Score { get; set; }
+
+        public string Class { get; set; }
+
+        public DateTime Timestamp { get; set; }
+
+        public PPagination PPagination { get; set; }
```

Comment:
Should we initialize it with `new PPagination`?

Reply:
done